src/viam/app/data_client.py

```diff
@@ -174,33 +174,48 @@ async def tabular_data_by_filter(
         return data
 
     async def binary_data_by_filter(
-        self,
-        filter: Optional[Filter] = None,
-        dest: Optional[str] = None,
+        self, filter: Optional[Filter] = None, dest: Optional[str] = None, include_file_data: bool = True, num_files: Optional[int] = None
     ) -> List[BinaryData]:
         """Filter and download binary data.
 
         Args:
-            filter (viam.proto.app.data.Filter): Optional `Filter` specifying binary data to retrieve. No `Filter` implies all binary data.
-            dest (str): Optional filepath for writing retrieved data.
+            filter (Optional[viam.proto.app.data.Filter]): Optional `Filter` specifying binary data to retrieve. No `Filter` implies al
+                binary data.
+            dest (Optional[str]): Optional filepath for writing retrieved data.
+            include_file_data (bool): Boolean specifying whether to actually include the binary file data with each retrieved file. Defaults
+                to false (i.e., only the files' metadata is returned).
+            num_files (Optional[str]): Number of binary data to return. Passing 0 returns all binary data matching the filter no matter.
+                Defaults to 100 if no binary data is requested, otherwise 10. All binary data or the first `num_binary_data` will be
```

Comment:
What is `num_binary_data` referenced here?

Reply:
ah good catch that is supposed to say num_files